modules/ocf_desktop/files/xsession/Xsession

```diff
@@ -114,7 +114,10 @@ xset m 0 0
 sleep 8 && /usr/local/bin/fix-audio &
 
 # Auto logout inactive users
-/usr/bin/xautolock -locker /usr/local/bin/auto-lock -time 7 -noclose &
+# killer is called killtime minutes after locker is called
+# any user input including shaking the mouse then not clearing the lockscreen is considered user input
```

Comment:
So if anyone moves the mouse to see if they are logged in, and then leaves it then the timer resets? If so, the timer on this should be much much shorter than 90 minutes total, because that will happen a lot.

Maybe a different method should be used if this is what it does, like a background script that logs the user out after a certain amount of inactivity? Seems like that would be pretty simple at least, and could be forked off `auto-lock` once it actually locks the screen.